docs/conventions/print.md

```diff
@@ -0,0 +1,232 @@
+# How to handle print css rules in ECL
+
+## Adopted solution
+
+- Print CSS and screen CSS are kept into separated files
+    ```
+    <head>
+      ...
+      <link rel="stylesheet" type="text/css" href="ecl-<preset>.css"  media="screen">
+      <link rel="stylesheet" type="text/css" href="ecl-<preset>-print.css" media="print">
+    </head>
+    ```
+
+- No global reset CSS for print
+
+- Print CSS is applied per component, using component's classes. Only needed rules are reported, to keep print CSS as small as possible.
+    ```
+    (screen css)
+    .my-component {
+      background-color: <yellow>;
+      color: <grey>;
+      display: flex;
+      font: <font-m>;
+      width: 100%;
+    }
+    ```
+    ```
+    (print css)
+    .my-component {
+      color: black;
+      font: <font-print-m>;
+      width: 100%;
+    }
+    ```
+    
+- Design tokens are added for print display (typography, spacing, ...)
+
+Below are the discussed options, for the record
+
+
+## File structure
+
+Mainly anwser this question: should we group all CSS in one file or split it in several files?
+
+### Option 1: One CSS for everything
+
+It means that screen and print CSS rules would be in a single SCSS file.
```

Comment:
not necessarily, we could have 2 SCSS files and still get one CSS file in the end :°